nomos-blend/message/src/crypto.rs

```diff
@@ -14,7 +14,7 @@ use serde_big_array::BigArray;
 
 pub const KEY_SIZE: usize = 32;
 
-#[derive(Clone)]
+#[derive(Debug, Clone, Serialize, Deserialize)]
```

Comment:
I guess this is fine for now and for testing, but we don't really want to serialize/deserialize a private key like that in production.